ui/receive_page.go

```diff
@@ -329,6 +329,7 @@ func (pg *receivePage) handle() {
 
 	if common.subPageBackButton.Button.Clicked() {
 		common.changePage(*common.returnPage)
+		common.subPageBackButton.Icon = common.icons.navigationArrowBack
```

Comment:
This looks like it should be in the onClose() method for the receive page. Same as the send page.